src/components/modal/Modal.scss

```diff
@@ -133,3 +135,28 @@
     z-index: 89;
     display: flex;
 }
+
+//Responsive support - triggers fullscreen mode after reaching $small-medium-screen width
+.is-responsive-web {
+    @include breakpoint($small-medium-screen) {
+        .modal {
+            flex-direction: column;
+            padding: 0;
+        }
+
+        .modal-dialog {
+            min-height: 100%;
+            padding: 22px 16px 12px 16px;
```

Comment:
Would it make sense to extract those values as variables?

Reply:
Personally, I feel like we'd only need to do this if it's being used somewhere else and/or stated specifically in a styling guide (correct me if there is one). Otherwise it'd seem like a lot of extra work for little benefit